### ISSUE TITLE
As the Deschutes Open Forest POC, I want to review and have Christmas tree permit content updated so that I know the information is correct before the season starts.

### ISSUE THREAD

**Issue by lisamcmichael** [USDAForestService/fs-open-forest-platform#808]
## Notes
See EPIC #799
Word doc on Pinyon: https://usfs.box.com/s/djr5fxga5myjb6gfd3c4h2nph8zhun3d

## Acceptance Criteria
- [x] POC reviewed and approved content
- [x] POC approved any backlog content ( if there is any for next season) 
- [x] The Forest's content has been updated based on Forest POC review.

## Tasks
- [x] Start scheduling content review with POC
- [x] Complete a demo with the forest, captured input
- [x] Review findings with PO

- [x] *Implement changes*
- [x] WHEN TO CUT YOUR TREE: Update Season Dates: Nov 18, 2019 - Dec 25, 2019
- [x] WHERE TO CUT YOUR TREE: Add "Cut trees only within the national forest boundary and respect surrounding private property." *Before** "No cutting or removing trees within:"
- [x] Under identifying tree species: [include photo of Lodgepole Pine]
- [x] HOW TO PLAN YOUR TRIP: **Remove** As a reminder, weather conditions at higher elevations can be dramatically different from local conditions and the public should be prepared for cold and unpredictable weather. Additionally, most forest roads are not maintained for winter driving, so snow patches or debris may be encountered.
- [x] CONTACT US: **Link** "local vendors" to the hyperlink provided: "You can buy Christmas tree permits in person at the following Deschutes National Forest ranger district offices or at local vendors."
https://www.fs.usda.gov/detail/deschutes/passes-permits/forestproducts/?cid=fsbdev3_035887
- [x] CONTACT US: under Deschutes National Forest Headquarters
Remove "Lisa Clark". Change phone number to (541) 383-5300
- [x] CONTACT US: under Deschutes National Forest Headquarters (title), add: Bend-Fort Rock Ranger District Office (note, they both share the same address)
- [x] CONTACT US: After Deschutes National Forest Headquarters show Crescent Ranger District Office, 136471 Hwy 97 North / PO Box 208, Crescent, OR 97733
 (541) 433-3200
- [x] CONTACT US: after Sisters Ranger District, **Add** Redmond Air Center. 1740 SE Ochoco Way, Redmond, OR 97756, (541) 504-7200
- [x] CONTACT US: **Remove** Ochoco National Forest

- [x] Forest POC has been informed of updates.
- [x] PO approved

## Definition of Done
- [ ] Pull requests meet technical definition of done
- [ ] Compare finished design with mockup
- [ ] Usability tested with forest

**Comment by lisamcmichael** [comment]
Hi @ASprinkle There seems to be some "unique" requests from this forest, noted above, about the Contact Us section. I'll put time on our calendars to review.

Folder on Box has Word version:
https://usfs.box.com/s/bynzqcipkakk1b1vifti3ajun885rhe6

**Comment by lisamcmichael** [comment]
@ASprinkle When you can, scan through this to see if I've captured our changes. The above "tasks" are at the end of the word document too. Thanks.
https://usfs.box.com/s/djr5fxga5myjb6gfd3c4h2nph8zhun3d

**Comment by ASprinkle** [comment]
Hi Lisa, I reviewed an commented in Pinyon "I sent a quick email to Brian Tany regarding the Ochoco info.  I anticipate it will be confusing to customers to see Ochoco info, and to Ochoco employees if someone calls them with Open Forest questions, since they are not a pilot forest.  Also, it looks like there are two "Helpful Cutting Tips" sections in the word doc. Can you remove one"

**Comment by lisamcmichael** [comment]
That sounds good @ASprinkle. Also, there is a newer version to review (cleaned up the 'bugs').

**Comment by ASprinkle** [comment]
Blocked while we wait for response back from @aaronburk regarding the potential addition of the Ochoco National Forest per Brian Tandy's request and information sent previous: 
Email from Brian Tandy:
"They are managed together in that they are virtually identical with the same (cutting rules and specifications, etc.).  We use the same “Guidelines for collecting your Christmas tree” flyer that contains cutting rules and specs, etc. for both Forests (see attached).  I think you could copy the Deschutes Open Forest webpage and use it intact, with some minor changes (Office addresses and phone numbers) for the Ochoco NF.   I am the point of contact for both Forests as I am the Forest Products Program Manager (that includes timber and SPF) for both Forests.   Also, we don’t use maps but rely on the flyer on where folks can cut their trees.  If a particular unit does use a map they are generalized maps on where folks can find particular species of trees to cut."

**Comment by lisamcmichael** [comment]
@ASprinkle Here are Brian's comments about Xmas Tree permit usage across forests:

_The hard-copy Christmas tree permits sold by the Deschutes and Ochoco NFs and our Central Oregon vendors, would be valid across any other Forest across the nation.  The reason for this, is that the hard-copy Christmas tree permit (see attached from last year) specifies only the “USDA Forest Service” and does not specify any specific National Forest or Ranger District.  To avoid confusion for the public, I would expect the e-Christmas tree permits to be exactly the same as the hard-copy permits.  I understand that only certain Forests are in the Pilot program, however, once an e-permit is sold, technically it should be good for any National Forest.  I assume that identifying the Forest that the e-permit is purchased from helps funnel those funds (FDDS) back to the Forest from which the e-permit was purchased?_

**Comment by ASprinkle** [comment]
Unblocked. We know we will build the Deschutes NF as a separate unit. Waiting to hear back from the Ochoco NF Forest Supervisor with the request to be added to the system.

**Comment by ASprinkle** [comment]
@lisamcmichael per your question from the task box: 
@ASprinkle (not sure about this request): 
**From Brian: Please include ... the “Crooked River National Grassland” on the Ochoco page.**

We will not include the Ochoco NF & Crooked River National Grassland contact information on the Deschutes Open Forest Page.

**Comment by lisamcmichael** [comment]
Word Version of Content Review on Box
https://usfs.box.com/s/djr5fxga5myjb6gfd3c4h2nph8zhun3d

**Comment by lisamcmichael** [comment]
Removed Ochoco @ASprinkle, so it's up-to-date. Pending is if Ochoco will be in the Pilot.

**Comment by ASprinkle** [comment]
Ochoco is confirmed - it will be part of the pilot

**Comment by briandavidson** [comment]
@ASprinkle can we move this one back to in progress? I jumped the gone on moving it to awaiting acceptance -- this one also requires the seeder to run.

**Comment by ASprinkle** [comment]
@briandavidson Is there any more work being done on this story? Are we only waiting for the seeder to run before we can push the changes from this story? If so, I'd say we move this to Blocked if completing this story requires the completion of another task.  

Perhaps it's worth creating a new issue : "Run Seeder File updates" so that we can follow that. thoughts?
